source/_integrations/openuv.markdown

```diff
@@ -20,14 +20,6 @@ The `openuv` integration displays UV and Ozone data from [openuv.io](https://www
 To generate an API key,
 [simply log in to the OpenUV website](https://www.openuv.io/auth/google).
 
-<div class='note warning'>
-Beginning February 1, 2019, the "Limited" plan (which is what new users are
```

Comment:
While a warning might be a bit much, this is still true and should be still mentioned.